test/unit/org/apache/cassandra/repair/RepairJobTest.java

```diff
@@ -350,13 +351,23 @@ public void testCreateStandardSyncTasksWithPushRepair()
                                                                                     true,
                                                                                     false,
                                                                                     PreviewKind.ALL));
-        assertThat(tasks).hasSize(1);
+        assertThat(tasks).hasSize(2);
```

Comment:
do I understand correctly that without the fix there would have been an additional task for addr2<->addr3?

Reply:
right.